R/helpers.R

```diff
@@ -164,3 +164,40 @@ mergeCountMatrices <- function(cms, transposed=FALSE, ...) {
   }
   return(res)
 }
+
+#' Check for installed package(s)
```

Comment:
I'd remove this line

Do the following:

```
#'  Check whether a package is installed and suggest how to install from either CRAN or Bioconductor
#'
#' @param pkgs character Package name(s)
```

You'll see that the roxygen2 docs will render correctly

Reply:
Implemented